src/htcondor_es/convert_to_json.py

```diff
@@ -588,12 +588,12 @@ def convert_to_json(ad, cms=True, return_dict=False, reduce_data=False):
         else:
             camp2 = ttype
 
-        result['CMS_JobType'] = str(ad.get('CMS_JobType', 'Analysis' if analysis else 'Unknown'))
+        result['CMS_JobType'] = str(ad.get('CMS_JobType' 'Unknown'))
```

Comment:
You missed a comma "," here. How about making it backwards compatible for the moment (until CRAB jobs start showing up with this CMS_JobType ad too)?

Reply:
We're stupid. This was already correct (and backwards compatible) as it was. If `'CMS_JobType'` is filled, it's already taken. Only if it's not filled we overwrite it with `"Analysis"`.

Reply:
LoL, you're right!